extensions/opentelemetry-resteasy-reactive/pom.xml

```diff
@@ -0,0 +1,22 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <parent>
+        <artifactId>quarkus-extensions-parent</artifactId>
+        <groupId>io.quarkus</groupId>
+        <version>999-SNAPSHOT</version>
+        <relativePath>../pom.xml</relativePath>
+    </parent>
+    <modelVersion>4.0.0</modelVersion>
+
+    <artifactId>quarkus-opentelemetry-resteasy-reactive-parent</artifactId>
```

Comment:
I assume ppl will have to know about this extension to activate tracing on resteasy reactive, right?
I wonder if this could live inside the main quarkus-opentelemetry extension. We have other SPI activations in there.

Reply:
Nope, that's the whole point of the conditional extensions.

It will get activated automatically when OTel and RR extensions are added by the user

Reply:
See https://quarkus.io/guides/conditional-extension-dependencies

Reply:
Thanks @geoand. Now I get it.